test/ngAria/ariaSpec.js

```diff
@@ -616,10 +616,35 @@ describe('$aria', function() {
   describe('tabindex', function() {
     beforeEach(injectScopeAndCompiler);
 
-    it('should attach tabindex to role="checkbox", ng-click, and ng-dblclick', function() {
+    it('should not attach to native controls', function() {
+      var element = [
+        $compile("<button ng-click='something'></button>")(scope),
+        $compile("<a ng-href='#/something'>")(scope),
+        $compile("<input ng-model='val'>")(scope),
+        $compile("<textarea ng-model='val'></textarea>")(scope),
+        $compile("<select ng-model='val'></select>")(scope),
+        $compile("<details ng-model='val'></details>")(scope)
+      ];
+      expectAriaAttrOnEachElement(element, 'tabindex', undefined);
+    });
+
+    it('should not attach to rando ng-model elements', function() {
```

Comment:
1. "rando" is what I would say verbally...it was intentional, but could be changed as it is colloquial.
2. `tabindex` is only useful if it has interaction events as well, for which ngModel alone is not responsible. That's more of a job for ngClick (or similar), which does add tabindex. The whole problem with ngAria was that it was way too liberal with adding tabindex. This PR fixes that.